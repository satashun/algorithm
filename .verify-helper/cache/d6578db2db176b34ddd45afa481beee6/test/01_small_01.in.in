8 20
0 4 15
1 1
1 2
1 7
0 6 4
0 3 5
0 2 8
1 1
1 2
1 3
0 6 11
0 7 33
1 1
1 2
1 3
0 3 20
1 1
1 2
1 3
1 4
