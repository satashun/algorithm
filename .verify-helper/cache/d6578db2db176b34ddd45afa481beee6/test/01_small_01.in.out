4 15
1 0
7 0
4 15
2 8
3 5
7 33
4 15
6 15
7 33
3 25
4 15
6 15
